### PR TITLE
Qe sso 7.4 5

### PR DESCRIPTION
**Pull request by pradeshm**

### CODE REVIEW

File: server/tests/src/test/java/org/infinispan/server/security/AuthenticationKeyCloakIT.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
 
 
    @ClassRule
-   public static KeyCloakServerRule KEYCLOAK = new KeyCloakServerRule("keycloak/infinispan-keycloak-realm.json");
+   public static KeyCloakServerRule KEYCLOAK = new KeyCloakServerRule("keycloak/infinispan-rhsso-realm.json","registry.redhat.io/rh-sso-7/sso74-openshift-rhel8:7.4-5");
```
Comment (tristantarrant) [inline server/tests/src/test/java/org/infinispan/server/security/AuthenticationKeyCloakIT.java:37]: If you want to be able to specify alternate images, you should have a way to override using a system property. See the DB tests for an example.

File: server/tests/src/test/resources/keycloak/infinispan-rhsso-realm.json [hunk 2]
```diff
@@ -0,0 +1,135 @@
+{
```
Comment (tristantarrant) [inline server/tests/src/test/resources/keycloak/infinispan-rhsso-realm.json:1]: This json was exported without secrets. You should use KeyCloak's export tool https://github.com/keycloak/keycloak-documentation/blob/master/server_admin/topics/export-import.adoc

**Comment by tristantarrant** [comment]
Closing this. Please reopen when it's ready.

**Comment by pradeshm** [comment]
Thank you for the pointers @tristantarrant I will reopen once it's ready!

**Pull request closed without merging**